### ISSUE TITLE
Is there anything wrong with the steps I train the transformer?

### ISSUE THREAD

**Issue by hjq133** [CompVis/taming-transformers#104]
Thanks for your great work !

I meet some problem when I train an unconditional transformer on IMagenet.

here is my config file , and I run it on 1 32G V100 GPU

```yaml
model:
  base_learning_rate: 0.00625
  target: taming.models.cond_transformer.Net2NetTransformer
  params:
    transformer_config:
      target: taming.modules.transformer.mingpt.GPT
      params:
        vocab_size: 16384
        block_size: 256
        n_layer: 48
        n_head: 24
        n_embd: 1536
    first_stage_config:
      target: taming.models.vqgan.VQModel
      params:
        ckpt_path: /mnt/lustre/huangjunqin/taming_transformer/logs/imagenet_vqgan_f16_16384/checkpoints/last.ckpt
        embed_dim: 256
        n_embed: 16384
        ddconfig:
          double_z: false
          z_channels: 256
          resolution: 256
          in_channels: 3
          out_ch: 3
          ch: 128
          ch_mult:
          - 1
          - 1
          - 2
          - 2
          - 4
          num_res_blocks: 2
          attn_resolutions:
          - 16
          dropout: 0.0
        lossconfig:
          target: taming.modules.losses.vqperceptual.DummyLoss
    cond_stage_config: __is_unconditional__

data:
  target: main.DataModuleFromConfig
  params:
    batch_size: 1
    wrap: false
    train:
      target: taming.data.imagenet.ImageNetTrain
      params:
        config:
          size: 256

```

I use one of your share _ckpt_ trained on Imagenet as my VQ model's pretrain.

when it starts training, the loss decreases, but only a few hundred iterations it converge to around 6.8. And when I check my image log. I found that the reconstruction image is fun, but the samples_nopix image's quality is getting worse, and the  sample_det image is always a picture of a certain color:

iter 2 sample no pix:
![samples_nopix_iter2](https://user-images.githubusercontent.com/37239015/132546777-5db0c4b4-74dc-400e-8f3e-2aadb2702e33.png)

iter 8 sample no pix:
![samples_nopix_iter8](https://user-images.githubusercontent.com/37239015/132546842-5180855b-e3f7-4ea5-b6b1-c04e5fc8f7c4.png)

iter 256 sample no pix:
![samples_nopix_iter256](https://user-images.githubusercontent.com/37239015/132546909-363a9ab1-8dc3-474d-a81f-69da6c13d13d.png)

___________________

iter 128 sample det

![samples_det_gs-000128_e-000000_b-000128](https://user-images.githubusercontent.com/37239015/132547188-9d30b0cb-783c-4d30-a6f4-69fa952e43b2.png)

iter 750 sample det
![samples_det_gs-000750_e-000000_b-000750](https://user-images.githubusercontent.com/37239015/132547223-0bff1b00-24be-4b5b-b0bc-12b1a37a2de1.png)

_________________________

Is there something wrong ? Thanks for any help !

**Comment by order-a-lemonade** [comment]
hi, i meet the same problem, have you solved it?

**Comment by order-a-lemonade** [comment]
> hi, i meet the same problem, have you solved it?

i find the reason is my edition of source code. so just use the source code is ok.
